ResearchKit/Common/ORKChoiceViewCell.m

```diff
@@ -292,7 +292,10 @@ - (UIButton *)detailTextIndicator {
     if (_detailTextIndicator == nil) {
         _detailTextIndicator = [UIButton buttonWithType:UIButtonTypeInfoDark];
         CGFloat inset = (DetailTextIndicatorTouchTargetWidth - DetailTextIndicatorImageWidth) / 2;
+#pragma clang diagnostic push
```

Comment:
clever idea

Reply:
I meant to explain in the PR description: there are _many_ compiler warnings in ResearchKit about edge insets, like this:

> contentEdgeInsets' is deprecated: first deprecated in iOS 15.0 - This property is ignored when using UIButtonConfiguration

I'm skeptical it's worth the effort to address all of these. In my experience addressing these in MyDataHelps, it can take a fair amount of tinkering to migrate to using UIButton.Configuration and get the same appearance/behavior, in some cases I opted to just rewrite the entire button from scratch.

And I think we should be safe to leave these as-is, from what I know it should only break if we start using UIButton.Configuration without updating these deprecated properties, which shouldn't happen within ResearchKit.